criu/util.c

```diff
@@ -1367,6 +1367,9 @@ void rlimit_unlimit_nofile(void)
 {
 	struct rlimit new;
 
+	if (opts.uid)
```

Comment:
imho: this check should be out of this function. I mean it should be in a place where this function is called and we need to support it with a comment that explains why we can skip it if we don't have enough permissions.